plugins/ledger/ledger.py

```diff
@@ -171,7 +177,12 @@ def perform_hw1_preflight(self):
                 self.dongleObject.verifyPin(pin)
 
             try:
-                self.dongleObject.getWalletPublicKey("44'/145'/0'/0/0", showOnScreen=False, cashAddr=True)
+                try:
+                    self.dongleObject.getWalletPublicKey("44'/145'/0'/0/0", showOnScreen=False, cashAddr=True)
+                except BTChipException as e:
+                    # This call fails on HW1/Nano with a specific error, due to it not supporting cashaddr
+                    if (e.sw != 0x6b00):
+                        raise e
```

Comment:
I think we could remove this change as well.
This was used for the detection of the HW1/Nano but does not do anything now.

Reply:
This piece of code is not used to detect HW1 but to check if the software library supports CashAddr. Either we handle the exception or we need to use a different method to detect CashAddr support in `btchip-python`